Use commonjs syntax instead of sprockets syntax

This will make it possible for bundling with standars nodejs tools like rollup
instead of blade, which is not packaged.

See node-turbolinks for a similar patch.

--- a/actionview/app/assets/javascripts/rails-ujs.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs.coffee
@@ -1,9 +1,17 @@
-#= require ./rails-ujs/BANNER
-#= export Rails
-#= require_self
-#= require_tree ./rails-ujs/utils
-#= require_tree ./rails-ujs/features
-#= require ./rails-ujs/start
+require './rails-ujs/BANNER'
+module.exports = @Rails;
+window.Turbolinks = @Rails;
+require './rails-ujs/utils/ajax'
+require './rails-ujs/utils/csp'
+require './rails-ujs/utils/csrf'
+require './rails-ujs/utils/dom'
+require './rails-ujs/utils/event'
+require './rails-ujs/utils/form'
+require './rails-ujs/features/confirm'
+require './rails-ujs/features/disable'
+require './rails-ujs/features/method'
+require './rails-ujs/features/remote'
+require './rails-ujs/start'
 
 @Rails =
   # Link elements bound by rails-ujs
--- a/actionview/app/assets/javascripts/rails-ujs/utils/ajax.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/utils/ajax.coffee
@@ -1,6 +1,6 @@
-#= require ./csp
-#= require ./csrf
-#= require ./event
+require './csp'
+require './csrf'
+require './event'
 
 { cspNonce, CSRFProtection, fire } = Rails
 
--- a/actionview/app/assets/javascripts/rails-ujs/features/confirm.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/features/confirm.coffee
@@ -1,4 +1,9 @@
-#= require_tree ../utils
+require '../utils/ajax'
+require '../utils/csp'
+require '../utils/csrf'
+require '../utils/dom'
+require '../utils/event'
+require '../utils/form'
 
 { fire, stopEverything } = Rails
 
--- a/actionview/app/assets/javascripts/rails-ujs/features/disable.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/features/disable.coffee
@@ -1,4 +1,9 @@
-#= require_tree ../utils
+require '../utils/ajax'
+require '../utils/csp'
+require '../utils/csrf'
+require '../utils/dom'
+require '../utils/event'
+require '../utils/form'
 
 { matches, getData, setData, stopEverything, formElements } = Rails
 
--- a/actionview/app/assets/javascripts/rails-ujs/features/method.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/features/method.coffee
@@ -1,4 +1,9 @@
-#= require_tree ../utils
+require '../utils/ajax'
+require '../utils/csp'
+require '../utils/csrf'
+require '../utils/dom'
+require '../utils/event'
+require '../utils/form'
 
 { stopEverything } = Rails
 
--- a/actionview/app/assets/javascripts/rails-ujs/features/remote.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/features/remote.coffee
@@ -1,4 +1,9 @@
-#= require_tree ../utils
+require '../utils/ajax'
+require '../utils/csp'
+require '../utils/csrf'
+require '../utils/dom'
+require '../utils/event'
+require '../utils/form'
 
 {
   matches, getData, setData
--- a/actionview/app/assets/javascripts/rails-ujs/utils/csrf.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/utils/csrf.coffee
@@ -1,4 +1,4 @@
-#= require ./dom
+require './dom'
 
 { $ } = Rails
 
--- a/actionview/app/assets/javascripts/rails-ujs/utils/event.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/utils/event.coffee
@@ -1,4 +1,4 @@
-#= require ./dom
+require './dom'
 
 { matches } = Rails
 
--- a/actionview/app/assets/javascripts/rails-ujs/utils/form.coffee
+++ b/actionview/app/assets/javascripts/rails-ujs/utils/form.coffee
@@ -1,4 +1,4 @@
-#= require ./dom
+require './dom'
 
 { matches } = Rails
 
